src/System.Private.ServiceModel/src/project.json

```diff
@@ -21,7 +21,7 @@
     "System.Net.Primitives": "4.0.10",
     "System.Net.WebHeaderCollection": "4.0.0",
     "System.Net.WebSockets": "4.0.0",
-    "System.Net.WebSockets.Client": "4.0.0",
+    "System.Net.WebSockets.Client": "4.0.1-beta-24322-03",
```

Comment:
There's no api surface changes, so if we needed to we could role back our dependency (I made no product changes to enable this fix) and release a stable version which doesn't support WS on linux.